### DISCUSSION TITLE
ツール

### DISCUSSION THREAD

**Discussion by haruki256** [gBenkyous/inTime#44] [Ideas]
dbt 
devopsのための各ツール

**Comment by haruki256** [comment]
MySQLで使える無料でおすすめのデータ変換レイヤーとしては、以下のものがあります。

dbt：データ変換をSQLで記述し、テストやドキュメントを自動生成するツールです。MySQLに対応しており、オープンソースで無料で利用できます。
Pentaho Data Integration：データ変換をグラフィカルなインターフェースで設計し、実行するツールです。MySQLに対応しており、コミュニティ版は無料で利用できます。
Talend Open Studio：データ変換をJavaのコードとして生成し、実行するツールです。MySQLに対応しており、オープンソースで無料で利用できます。
これらのツールはそれぞれ特徴や利点が異なるので、ご自身のニーズに合わせて選択することが重要です。

**Reply by doranko** [reply]
@honmaharuki 
導入を検討する目的についても記載お願いいたします。
少々調べたところELT・ETLのうちTを担当する機能ということで、まずはそれらの検討から必要なのでは？と思いました。。
